modules/enet/doc_classes/NetworkedMultiplayerENet.xml

```diff
@@ -97,6 +97,13 @@
 				Returns the remote port of the given peer.
 			</description>
 		</method>
+		<method name="get_local_port" qualifiers="const">
```

Comment:
This will be reordered by `doctool`, no? It doesn't follow alphabetical ordering.